rules/api/aws_instance_invalid_ami.go

```diff
@@ -72,13 +72,18 @@ func (r *AwsInstanceInvalidAMIRule) Check(rr tflint.Runner) error {
 			continue
 		}
 
+		awsClient, err := runner.AwsClient(resource.Body.Attributes)
```

Comment:
You need to get the provider attribute explicitly like any other rule.

Reply:
Fixed. 821423f